### PR TITLE
refactor: change div tags to more appropriate alternatives 

### PR DESCRIPTION
**Pull request by ErickSharp**
Fixes no known issue.

## Summary of changes
Changes the tags of components to be more descriptive and to try and adhere to best practice.
## Screenshots(if applicable)
No applicable screenshots; no functionality of the website was altered.

### CODE REVIEW

File: src/components/projects/A32NX/A32NXProjHeader.tsx [hunk 1]
```diff
@@ -3,9 +3,9 @@ import BackgroundImage from '../../../assets/img/discord.jpg';
 import a32nx_wireframe from '../../../assets/svg/a32nx_wireframe.svg';
```
Comment (NathanInnes) [inline src/components/projects/A32NX/A32NXProjHeader.tsx:3]: This doesn't need to be renamed to specifics as it's already in a folder called A32NX

Comment (ErickSharp) [reply src/components/projects/A32NX/A32NXProjHeader.tsx:3]: Hmm, maybe we could change it to something else. I still don't think that "Hero" makes sense, either.

Comment (NathanInnes) [reply src/components/projects/A32NX/A32NXProjHeader.tsx:3]: But that's what they're called

Comment (ErickSharp) [reply src/components/projects/A32NX/A32NXProjHeader.tsx:3]: yes yes you are correct, thank you for that; reverting back to 4645863

**Review by NathanInnes — APPROVED** [review]

**Pull request merged**